### ISSUE TITLE
Update demographic data form to be more international-aware

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#796]
# TL;DR
The current demographics questionnaire is confusing/not relevant to people outside the united states; there are higher and lower-effort ways we can mitigate this! Will require consult with experts/researchers who do data collection internationally. MBAH brought this critique and would like a better version! 

# Narrative
"As a User, I would like to see questions in the demographic survey that make sense for people in my country, or at least signal when they aren't relevant."

<details>
<summary>Feature Request fields</summary>
<p>
**With this `details` wrapper, you can just use this ticket to record your findings.
When you're ready to turn this into an implementation request, delete the scoping label and
this wrapper, leaving only the filled out information.**

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests for new feature
- [ ] QA on new feature
- [ ] Sign off from Kim

# Implementation Notes
Leave important notes/suggestions here for implementation (important files, suggested approaches, etc.)

</p>
</details>
